src/unasync/__init__.py

```diff
@@ -18,6 +18,9 @@
     "__aiter__": "__iter__",
     "__anext__": "__next__",
     "asynccontextmanager": "contextmanager",
+    "AsyncIterable": "Iterable",
+    "AsyncIterator": "Iterator",
+    "AsyncGenerator": "Generator",
```

Comment:
Do we need to spell out those cases here? It looks like they'll be handled automatically by the code below

Reply:
Have to have these here otherwise the custom-class logic transforms them like so:

```
typing.AsyncIterable[bytes] -> typing.SyncIterable[bytes]
```

should be `typing.Iterable[bytes]` post-transform.